File: CPP/Program/Anggota.cpp
//import library

#include <iostream>
#include <string>

//using standard namespace 
using namespace std;

//class declaration. for C++, the first letter doesnt have to be
//capitalized, but its better so we can distinguish it with the other types
class Anggota
{
    //private atribut
    private:
        int id;
        string name;
        string nama_bidang;
        string nama_partai;
    //public methods    
    public:
        //constructor//
        
        //contructor. take note that it doesnt have any return type
        Anggota()
        {
            //set name and gender to default value
            this->id = '0';
            this->name = "";
            this->nama_bidang = "";
            this->nama_partai = "";
        }

        //another contructor with parameter. this one will be called if the
        //object instanced with the given parameter
        Anggota(int id,string name, string nama_bidang, string nama_partai)
        {
            //"this" object's attributes will be set with the given parameter attributes
            this->id = id;
            this->name = name;
            this->nama_bidang = nama_bidang;
            this->nama_partai = nama_partai;
        }

        //getter and setter//

        //get id
        int get_id(){
            return this->id;
        }

        //set id
        void set_id(int id){
            this->id = id;
        }

        //get name
        string get_name(){
            return this->name;
        }

        //set name
        void set_name(string name){
            this->name = name;
        }

        //get nama_bidang
        string get_nama_bidang(){
            return this->nama_bidang;
        }

        //set nama_bidang
        void set_nama_bidang(string nama_bidang){
            this->nama_bidang = nama_bidang;
        }

        //get nama_partai
        string get_nama_partai(){
            return this->nama_partai;
        }

        //set nama_partai
        void set_nama_partai(string nama_partai){
            this->nama_partai = nama_partai;
        }


        //destructors//

        //default destructor. leave it blank for now
        ~Anggota(){
            
        }
};  

File: Java/Program/Anggota.java


// class declaration. For Java, the first letter MUST BE capitalized.
public class Anggota{
    // Private attributes.
        private int id;
        private String name;
        private String nama_bidang;
        private String nama_partai;
        
    /* Constructors. */
    // Constructor. Take note that it doesn't have any return type.
    public Anggota(){
        this.id = '0';
        this.name = "";
        this.nama_bidang = "";
        this.nama_partai = "";
        
    }

    // Another constructor with parameter.
    public Anggota(int id, String name, String nama_bidang, String nama_partai){
        this.id = id;
        this.name = name;
        this.nama_bidang = nama_bidang;
        this.nama_partai = nama_partai;
    }
    /* Getter and Setter. */
    // Get gender.
    public int getId(){
        return this.id;
    }

    // Set gender.
    public void setId(int id){
        this.id = id;
    }

    // Get name.
    public String getName(){
        return this.name;
    }
    // Set name.
    public void setName(String name){
        this.name = name;
    }

    // Get name.
    public String getNamaBidang(){
        return this.nama_bidang;
    }
    // Set name.
    public void setNamaBidang(String nama_bidang){
        this.nama_bidang = nama_bidang;
    }

    // Get name.
    public String getNamaPartai(){
        return this.nama_partai;
    }
    // Set name.
    public void setNamaPartai(String nama_partai){
        this.nama_partai = nama_partai;
    }

}